apps/vocabulary/v2Processor.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"io/ioutil"
+
+	"github.com/golang/protobuf/proto"
+
+	openapi_v2 "github.com/googleapis/gnostic/openapiv2"
+)
+
+func readDocumentFromFileWithNameV2(filename string) (*openapi_v2.Document, error) {
+	data, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return nil, err
+	}
+	document := &openapi_v2.Document{}
+	err = proto.Unmarshal(data, document)
+	if err != nil {
+		return nil, err
+	}
+	return document, nil
+
+}
+
+func processDocumentV2(document *openapi_v2.Document, schemas map[string]int, operationId map[string]int, names map[string]int, properties map[string]int) {
```

Comment:
Not sure what the standard is for Go, so I'm just wondering if these arguments should break down to 1 arg per line(since the line is so long).

Reply:
here and for the other methods with many arguments

Reply:
Running gofmt -w . in the directory will “do the right thing”.